Source/WebCore/css/CSSFontFace.cpp

```diff
@@ -153,7 +153,7 @@ void CSSFontFace::setFamilies(CSSValueList& family)
     RefPtr oldFamily = std::exchange(m_families, &family);
     mutableProperties().setProperty(CSSPropertyFontFamily, family);
 
-    iterateClients(m_clients, [&](CSSFontFaceClient& client) {
+    iterateClients(m_clients, [this, protectedThis = Ref { *this }](CSSFontFaceClient& client) {
```

Comment:
The rules forbid capturing `this` fwiw. You should always use `protectedThis` instead.

Reply:
I see. I think we should update https://github.com/WebKit/WebKit/wiki/Safer-CPP-Guidelines#when-passing-an-object-to-non-trivial-function-hold-a-smart-pointer-to-this-object-on-the-stack

The right example shows:
```
// Keep `this` alive.
doAsyncWork([this, protectedThis] {
    foo(); // Member function on `this`.
    bar(); // Member function on `this`.
});

```

Reply:
Oh, capturing `this` is fine when there is a `protectedThis`. When we capture `WeakPtr` instead is when we need to avoid capturing `this`.